__tests__/utils/fhir.test.ts

```diff
@@ -158,6 +163,85 @@ const TEST_MEASURE_BUNDLE_WITH_EXPANSION: fhir4.Bundle = {
   ]
 };
 
+const RESOURCE_WITH_NO_SUMMARY: fhir4.Resource = {
+  resourceType: 'Prodcedure'
```

Comment:
Misspelling. purposeful or not? It would be good to have a test with a resource without a primary code path entry, ie. MeasureReport, but does have an `id`.

Reply:
I added a test with a resource without a primary code path entry.